modules/data/user.py

```diff
@@ -0,0 +1,84 @@
+"""Users management"""
+from modules.data.db_manager import DbManager
+from modules.data.pending_post import PendingPost
+
+
+class User():
+    """Class that represents a user
+    """
+    def __init__(self, user_id):
+        self.user_id = user_id
+
+    @property
+    def is_pending(self) -> bool:
+        """Checks if the user has a post already pending or not
+
+        Returns:
+            bool: whether the user has a post already pending or not
+        """
+        if PendingPost.from_user(self.user_id):
+            return True
+        return False
+
```

Comment:
```suggestion
        return bool(PendingPost.from_user(self.user_id))
```

this should work too, but make a quick check